src/components/Persona/Persona.scss

```diff
@@ -64,66 +77,66 @@ $ms-color-initials-darkRed:     #b91d47;
 }
 
 .ms-Persona-initials {
-  color: $ms-color-white;
-  font-size: $ms-font-size-l;
-  font-family: $ms-font-family-light;
-  line-height: 48px;
-
-  &.ms-Persona-initials--lightBlue {
-    background-color: $ms-color-initials-lightBlue;
-  }
-  &.ms-Persona-initials--blue {
-    background-color: $ms-color-initials-blue;
-  }
-  &.ms-Persona-initials--darkBlue {
-    background-color: $ms-color-initials-darkBlue;
-  }
-  &.ms-Persona-initials--teal {
-    background-color: $ms-color-initials-teal;
-  }
-  &.ms-Persona-initials--lightGreen {
-    background-color: $ms-color-initials-lightGreen;
-  }
-  &.ms-Persona-initials--green {
-    background-color: $ms-color-initials-green;
-  }
-  &.ms-Persona-initials--darkGreen {
-    background-color: $ms-color-initials-darkGreen;
-  }
-  &.ms-Persona-initials--lightPink {
-    background-color: $ms-color-initials-lightPink;
-  }
-  &.ms-Persona-initials--pink {
-    background-color: $ms-color-initials-pink;
-  }
-  &.ms-Persona-initials--magenta {
-    background-color: $ms-color-initials-magenta;
-  }
-  &.ms-Persona-initials--purple {
-    background-color: $ms-color-initials-purple;
-  }
-  &.ms-Persona-initials--black {
-    background-color: $ms-color-initials-black;
-  }
-  &.ms-Persona-initials--orange {
-    background-color: $ms-color-initials-orange;
-  }
-  &.ms-Persona-initials--red {
-    background-color: $ms-color-initials-red;
-  }
-  &.ms-Persona-initials--darkRed {
-    background-color: $ms-color-initials-darkRed;
-  }
+	color: $ms-color-white;
```

Comment:
It looks like this converts spaces to tabs. Generally we use 2 spaces per indent. Could you revert this section's spacing adjustments?